docs/content/preview/develop/quality-of-service/transaction-priority.md

```diff
@@ -218,7 +220,7 @@ The `yb_get_current_transaction_priority` function shows the transaction priorit
     (1 row)
     ```
 
-   As seen from the resulting _High priority transaction_ value above, it is randomly chosen between the lower and upper bound.
+   As seen from the resulting _High priority transaction_ value in the preceding example, the transaction priority is randomly chosen between the lower and upper bound.
```

Comment:
It looks like the second `set yb_transaction_priority_lower_bound` line should be setting the upper bound, too, not the lower bound a second time.

```suggestion
   The transaction priority is randomly chosen between the lower and upper bound.
```